pkg/cmd/codespace/list.go

```diff
@@ -50,8 +52,26 @@ func newListCmd(app *App) *cobra.Command {
 }
 
 func (a *App) List(ctx context.Context, opts *listOptions, exporter cmdutil.Exporter) error {
+	if (opts.orgName != "" || opts.userName != "") && opts.repo != "" {
```

Comment:
nit/ 🎨  -- I think this condition would read more naturally if flipped:

```go
// if repoName is provided, we don't accept orgName or userName
if opts.repo != "" && (opts.orgName != "" || opts.userName != "") 
```

but that may just be me, so feel free to ignore :-)